Update fluent-plugin-loki documentation URLs (#3880)
grafana/loki: docs/sources/clients/fluentd/_index.md

## Docker Image

> There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/main/clients/cmd/fluentd/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.

Reviewer: I find this paragraph confusing.  Here's my suggested rewrite, given my questionable understanding of what was meant to be conveyed.

```suggestion
The Docker image `grafana/fluent-plugin-loki:master` contains [default configuration files](https://github.com/grafana/loki/tree/main/clients/cmd/fluentd/docker/conf). By default, fluentd containers use that default configuration. You can instead specify your `fluentd.conf` configuration file with a `FLUENTD_CONF` environment variable.
```

### url

> The url of the Loki server to send logs to.  When sending data the publish path (`../api/loki/v1/push`) will automatically be appended.

Reviewer: Please also update this prose, even though your PR did not originally touch this prose.

> ### url
> 
> The url of the Loki server to send logs to.  When sending data the publish path (`../api/loki/v1/push`) will automatically be appended.

Reviewer: Please also update this prose, even though your PR did not originally touch this prose.

```suggestion
### URL

The URL of the Loki server to send logs to.  When sending data, the publish path (`../api/loki/v1/push`) will automatically be appended.
```
